runtime/astar/src/lib.rs

```diff
@@ -888,6 +892,34 @@ impl pallet_xc_asset_config::Config for Runtime {
     type WeightInfo = pallet_xc_asset_config::weights::SubstrateWeight<Self>;
 }
 
+parameter_types! {
+    pub MessageQueueServiceWeight: Weight =
+        Perbill::from_percent(25) * RuntimeBlockWeights::get().max_block;
+    pub const MessageQueueMaxStale: u32 = 8;
+    pub const MessageQueueHeapSize: u32 = 128 * 1048;
```

Comment:
Suggestion - use `ConstU32<>` notation instead.

Reply:
`ConstU32<>` can unfortunatly not be used in `parameter_types!` macro. And I would keep those 2 type in the macro